common/dynamicconfig/shared_constants.go

```diff
@@ -68,6 +68,12 @@ const (
 	// dynamicRateLimitIncreaseStepSizeKey the amount the rate limit multiplier is increased when the system is healthy. should be between 0 and 1
 	dynamicRateLimitIncreaseStepSizeKey     = "rateIncreaseStepSize"
 	dynamicRateLimitIncreaseStepSizeDefault = 0.1
+	// dynamicRateLimitMultiMinKey is the minimum the rate limit multiplier can be reduced to
+	dynamicRateLimitMultiMinKey     = "rateMultiMin"
+	dynamicRateLimitMultiMinDefault = 0.5
```

Comment:
nit: A higher value like 0.8 will be safer initially I guess. There are cases where we know reducing the rate won't help a lot and I guess it's better to limit the impact in those cases.